src/editor.c

```diff
@@ -5017,8 +5017,9 @@ GeanyEditor *editor_create(GeanyDocument *doc)
 	editor->line_wrapping = get_project_pref(line_wrapping);
 	editor->scroll_percent = -1.0F;
 	editor->line_breaking = FALSE;
-
+	
 	editor->sci = editor_create_widget(editor);
+	sci_set_multipaste(editor->sci, SC_MULTIPASTE_EACH);
```

Comment:
This should go in `editor_create_widget()` along the other similar ones.

Reply:
My bad, I meant `create_new_sci()` which is the one having the other similar defaults

Reply:
That's where it is now.